src/format.rs

```diff
@@ -235,6 +235,23 @@ pub struct AuditsFile {
     pub trusted: TrustedPackages,
 }
 
+impl Tidyable for AuditsFile {
+    fn tidy(&mut self) {
+        self.audits.retain(|_, entries| {
+            entries.sort();
+            !entries.is_empty()
```

Comment:
Due to the repetition, I wonder whether it'd make sense to have `impl<K, V: Ord> Tidyable for SortedMap<K, Vec<V>>` and use that.